src/Games/brain-cals.js

```diff
@@ -0,0 +1,31 @@
+import getRandomNumber from '../randomCount.js';
+import theCoreOfGames from '../index.js';
+
+const brainRulls = () => {
+  const descriptionOfGame = 'What is the result of the expression?';
+
+  const randomNumberOne = getRandomNumber(1, 10);
+  const randomNumberTwo = getRandomNumber(1, 10);
+  const operator = ['+', '-', '*'];
+  const randomOperator = getRandomNumber(0, 2);
+  const randomNumber = `${randomNumberOne} ${operator[randomOperator]} ${randomNumberTwo}`;
+
+  let correctAnswer;
+  if (operator[randomOperator] === '+') {
```

Comment:
Maybe the switch is better suited for choosing operators?

Reply:
add switch, ok